packages/react-magma-dom/src/components/StyledButton/styles.ts

```diff
@@ -254,14 +298,14 @@ export function buildAfterBackground(props) {
   ) {
     return props.theme.colors.neutral;
   }
-  return props.theme.colors.neutral08;
+  return props.theme.colors.neutral100;
 }
 
 export function buildActiveBackground(props) {
   props = buildPropsWithDefaultButtonStyles(props);
 
   if (props.color === 'marketing') {
-    return lighten(0.2, props.theme.colors.pop04);
+    return lighten(0.2, props.theme.colors.secondary500);
```

Comment:
Just to triple check, what is the difference between `lighten 0.2` and `transparentize 0.2`-- are they interchangeable?

Reply:
Cool! I guess `lighten` still makes sense for the active state. Thanks